testdata/src/go.uber.org/receivers/receivers.go

```diff
@@ -95,18 +95,6 @@ func testCaller(dummy bool, i int) {
 	}
 }
 
-func (s S) testNonPointerRecv() {
-	_ = s.f
-}
-
-func (*S) testNamelessPointerRecv(i int) *int {
-	return &i
-}
-
-func (S) testNamelessNonPointerRecv(i int) *int {
-	return &i
-}
```

Comment:
These methods are removed from here, since they were unused. Instead, they are updated and used in `receivers-with-inference.go` test file.